macos/RNCProgressView.m

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#import "RNCProgressView.h"
+
+@implementation RNCProgressView
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+  if ((self = [super initWithFrame:frame])) {
+    self.maxValue = 1.0;
+    self.indeterminate = NO;
+
+    // Default track color from NSProgressIndicator.
+    self.trackTintColor = [RCTUIColor colorWithRed: 237/255.0 green:237/255.0 blue:237/255.0 alpha:1.0];
+  }
+  return self;
+}
+
+- (void)setProgressImage:(UIImage *)progressImage
+{
+  progressImage.capInsets = NSEdgeInsetsMake(0, progressImage.size.width-0.5, 0, 0);
+  progressImage.resizingMode = NSImageResizingModeStretch;
+  _progressImage = progressImage;
+}
+
+- (void)setTrackImage:(UIImage *)trackImage
+{
+  trackImage.capInsets = NSEdgeInsetsMake(0, 0, 0, trackImage.size.width-0.5);
```

Comment:
Same curiosity on why a half point over and should probably make it a const

Reply:
This file was brought over verbatim from [react-native-macos](https://github.com/microsoft/react-native-macos/blob/master/React/Views/RCTProgressView.m), so I feel like it would be better to refactor this in a separate PR. Unfortunately there doesn't seem to be anything in react-native-macos's history that indicates why this 0.5 offset exists.